packages/charts/src/charts/circular-chart/tooltip-item.component.tsx

```diff
@@ -20,7 +20,7 @@ const TooltipItem: FC<Props> = ({ data, theme = defaultTheme }) => {
   ) : (
     <Container>
       <Text {...tooltipSettings.labels.typography}>
-        {data.label}
+        {data.label.toString()}
```

Comment:
I suggest to change it just to ``${data.label}`` as for `null` invoking `toString` will cause runtime error.

Reply:
Yup you're right